From 53b989e4ce32c941d748ad33502600497595914a Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Wed, 2 Nov 2022 14:01:58 -0600
Subject: [PATCH] Remove spurious wheel build dependency

Wheel isn't a build dependency so it shouldn't be in setup_requires.
Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
[james.hilliard1@gmail.com: backport from upstream commit
53b989e4ce32c941d748ad33502600497595914a]
---
 setup.cfg | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.cfg b/setup.cfg
index 0abd270..2cb3e06 100644
--- a/setup.cfg
+++ b/setup.cfg
@@ -46,7 +46,7 @@ packages =
     dns.rdtypes.CH
 python_requires = >=3.7
 test_suite = tests
-setup_requires = setuptools>=44; wheel; setuptools_scm[toml]>=3.4.3
+setup_requires = setuptools>=44; setuptools_scm[toml]>=3.4.3
 
 [options.extras_require]
 DOH = httpx>=0.21.1; h2>=4.1.0; requests; requests-toolbelt
-- 
2.34.1

